### ISSUE TITLE
Components do not maximize height when used in (Fast-) template

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#3583]
My users and co-developers are asking me to create applications that adjust nicely to big traders screens, smaller laptop screens and custom window sizes.

Its normally not a problem to get `width` working. But `height` I can very often not get working when using either prebuilt or custom templates.

There is an example below. Any fix or workaround suggestion would be appreciated.

## No Template - works well

![image](https://user-images.githubusercontent.com/42288570/171833611-79550531-a028-479b-a6e1-d52a7c254a4a.png)

```python
import panel as pn
import pandas as pd
import hvplot.pandas

pn.extension(
  sizing_mode="stretch_width",
  # template="fast",
)

data = pd.DataFrame({
    "x": list(range(0,100)), "y": list(range(0,100))
})
table_pane = pn.widgets.Tabulator(
  data,
  show_index=False,
  theme="fast",
  height=500,
  sizing_mode="stretch_both",
  disabled=True,
)
component = pn.Column(
    table_pane, 
    pn.widgets.Button(name="download"), 
    sizing_mode="scale_both", 
    background="lightgray"
)

component.servable()
```

## Template - does not work well

![image](https://user-images.githubusercontent.com/42288570/171833454-5e5daeb8-f983-476a-a66c-f062cc96401f.png)

Please notice if I don't add the `heigth=500` the height will only be around 50!

```python
import panel as pn
import pandas as pd
import hvplot.pandas

pn.extension(
  sizing_mode="stretch_width",
  template="fast",
)

data = pd.DataFrame({
    "x": list(range(0,100)), "y": list(range(0,100))
})
table_pane = pn.widgets.Tabulator(
  data,
  show_index=False,
  theme="fast",
  height=500,
  sizing_mode="stretch_both",
  disabled=True,
)
component = pn.Column(
    table_pane, 
    pn.widgets.Button(name="download"), 
    sizing_mode="scale_both", 
    background="lightgray"
)

component.servable()
```

**Comment by MarcSkovMadsen** [comment]
## Template -  Workaround

If I set the `.bk-root` my self to something responsive it seems to work

![image](https://user-images.githubusercontent.com/42288570/171839320-4a5f66c4-7dd5-4e29-b61f-e1e51016ec18.png)

```python
import panel as pn
import pandas as pd
import hvplot.pandas

pn.config.raw_css.append(
  """
  .bk-root {
    height: calc( 100vh - 200px ) !important;
  }
  """
)

pn.extension(
  sizing_mode="stretch_width",
  template="fast",
)

data = pd.DataFrame({
    "x": list(range(0,100)), "y": list(range(0,100))
})
table_pane = pn.widgets.Tabulator(
  data,
  show_index=False,
  theme="fast",
  sizing_mode="stretch_both",
  disabled=True,
)
component = pn.Column(
    table_pane, 
    pn.widgets.Button(name="download"), 
    sizing_mode="scale_both", 
    background="lightgray"
)

component.servable()
```

**Comment by MarcSkovMadsen** [comment]
The workaround is a major improvement. Most of the time the natural app takes up exactly the window size and resizes when the window changes. It just made my apps 50% better to find the workaround.

**Comment by philippjfr** [comment]
@MarcSkovMadsen Could you see if you could contribute the fix? This seems important for 1.0.

**Comment by Hoxbro** [comment]
This has been fixed on the main branch. So I will close the issue. You're welcome to reopen if I have missed something. 

![image](https://user-images.githubusercontent.com/19758978/226550601-172a7572-704f-44e5-bad2-5cea8369c3d1.png)

``` python

import panel as pn
import pandas as pd

pn.extension(
    sizing_mode="stretch_width",
    template="fast",
)

data = pd.DataFrame({"x": list(range(0, 100)), "y": list(range(0, 100))})
table_pane = pn.widgets.Tabulator(
    data,
    show_index=False,
    theme="fast",
    sizing_mode="stretch_both",
    disabled=True,
)
component = pn.Column(
    table_pane,
    pn.widgets.Button(name="download"),
    sizing_mode="scale_both",
    background="lightgray",
)

component.servable()
```
